### PR TITLE
Implement config defaults

### PR DESCRIPTION
**Pull request by forgetso**

### CODE REVIEW

**Comment by forgetso** [comment]
- [ ] TODO: Obtain SS58_FORMAT and PAIR_TYPE from polkadot package

File: demos/client-example/src/config.ts [hunk 1]
```diff
@@ -17,37 +17,30 @@
...
+import { EnvironmentTypesSchema, NetworkNamesSchema } from '@prosopo/types'
+import { ProsopoClientConfigInput } from '@prosopo/types'
 
-const { REACT_APP_PROSOPO_CONTRACT_ADDRESS, REACT_APP_SUBSTRATE_ENDPOINT } = z
-    .object({
-        REACT_APP_PROSOPO_CONTRACT_ADDRESS: z.string(),
-        REACT_APP_SUBSTRATE_ENDPOINT: z.string(),
-    })
-    .parse(process.env)
+const getWeb2 = (): boolean | undefined => {
```
Comment (HughParry) [inline demos/client-example/src/config.ts:23]: Could be much shorter snippet. If we default to true (which is potentailly expected behaviour) then:

return !(process.env.REACT_APP_WEB2 === 'false')

---

if we require undefined then something like:

process.env.REACT_APP_WEB2 === 'true' ? true : process.env.REACT_APP_WEB2 === 'false' ? false : undefined;

File: dev/scripts/src/scripts/getNetworks.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import { selectableNetworks } from '@polkadot/networks'
+import { testParasRococoCommon } from '@polkadot/apps-config'
+async function run() {
+    // // Construct
+    // const provider = 'ws://localhost:9944'
+    // // const provider = 'wss://shiden.public.blastapi.io'
+    // const wsProvider = new WsProvider(provider)
+    // const api = await ApiPromise.create({ provider: wsProvider })
+
+    selectableNetworks.forEach((network) => {
```
Comment (HughParry) [inline dev/scripts/src/scripts/getNetworks.ts:23]: Can we be consistent on using functional methods vs for loops within files(sorry this one pretty nitpicky)

**Review by HughParry — CHANGES_REQUESTED** [review]

File: dev/scripts/src/scripts/getNetworks.ts (hunk 2 above)
Comment (forgetso) [reply dev/scripts/src/scripts/getNetworks.ts:23]: This wasn't meant to be in the PR. I've deleted it.

File: demos/client-example/src/config.ts (hunk 1 above)
Comment (forgetso) [reply demos/client-example/src/config.ts:23]: Changed

**Review by HughParry — APPROVED** [review]

**Pull request merged**